app/src/main/res/values/help.xml

```diff
@@ -6,38 +6,31 @@
 
 &lt;b&gt;Introduction and Quick Start&lt;/b&gt;&lt;br/&gt;
 Sky Map is your catalog of the night sky! Point your Android-powered device up, and you will see a map of the brightest
-stars, constellations, and planets in that part of the sky.&lt;br/&gt;Next time  what to identify a bright star or planet, use Sky Map!&lt;br/&gt;
+stars, constellations, and planets in that part of the sky.&lt;br/&gt;&lt;br/&gt;
 
 &lt;b&gt;Visible Layers&lt;/b&gt;&lt;br/&gt;
 Sky Map organizes the objects it shows you into layers.  Each layer can
 be toggled using the on-screen controls.  The layers are:&lt;br/&gt;
-      <span>&#8226;</span>Stars&lt;br/&gt;
-      <span>&#8226;</span>Constellations&lt;br/&gt;
-      <span>&#8226;</span>Messier Objects&lt;br/&gt;
-      <span>&#8226;</span>Planets&lt;br/&gt;
-      <span>&#8226;</span>Meteor Showers&lt;br/&gt;
-      <span>&#8226;</span>Right Ascension and Declination Grid&lt;br/&gt;
+      <span>&#8226;</span> Stars&lt;br/&gt;
+      <span>&#8226;</span> Constellations&lt;br/&gt;
+      <span>&#8226;</span> Messier Objects&lt;br/&gt;
+      <span>&#8226;</span> Planets&lt;br/&gt;
+      <span>&#8226;</span> Meteor Showers&lt;br/&gt;
+      <span>&#8226;</span> Right Ascension and Declination Grid&lt;br/&gt;
       <span>&#8226;</span> Horizon and cardinal points&lt;br/&gt;&lt;br/&gt;
       
 The brightest stars, constellations, and planets are labeled. 
 Planets display as images that are larger than will appear in the sky.
 Sky Map may show objects that are above the horizon line (represented as a straight line)
-that you may not be able to use, unless you live in an area that is  flat and without buildings.&lt;br/&gt;&lt;br/&gt;
+and are unable to see, unless your field of view is unobstructed.&lt;br/&gt;&lt;br/&gt;
 
 &lt;b&gt;On-screen controls&lt;/b&gt;&lt;br/&gt;
-Tap on the screen to show the on-screen controls. On the side of the screen are
-the toggles for the layers described above.  They are orange when the layers are
-enabled and white when disabled.&lt;br/&gt;
-
-The bottom of the screen displays zoom controls and a toggle to switch between
-automatic and manual mode.&lt;br/&gt;&lt;br/&gt;
+On the left side of the screen are the toggles for the layers described above. They are orange when
+enabled and white when disabled. Pinch the screen to zoom and and out. /&gt;
 &lt;b&gt;Navigation Modes&lt;/b&gt;&lt;br/&gt;
-Sky Map installs in Automatic Mode and uses the phone\'s sensors
-to display a map of the sky in the direction your phonw is pointing. By
-switching into Manual Mode, you can drag the map with your finger to explore the
-sky. To rotate the map left or right, move the trackball or D-pad. On multitouch phones
-you can also rotate the map with two fingers and use a pinch gesture to zoom in and out.
-&lt;br/&gt;&lt;br/&gt;
+      
+Sky Map installs in Automatic mode and automatically displays a map of the sky wherever the phone is pointed.
```

Comment:
To reduce the use of word "automatic", say, "Sky Map installs in Automatic Mode, which displays a map......."